data/constants/keycodes/keycodes_0.0.1.hjson

```diff
@@ -36,7 +36,9 @@
         "0x52C0/0x001F": {
             "define": "QK_LAYER_TAP_TOGGLE"
         },
-        // 0x52E0/0x001F - UNUSED
+        "0x52E0/0x001F": {
```

Comment:
this should apparently go in the 0.0.2 hjsoon (and appears it can be pretty much just this)

Reply:
So just to confirm I should create a file `data/constants/keycodes/keycodes_0.0.2.hjson` and move my modified range there?